server/db/Seeder.js

```diff
@@ -4,11 +4,35 @@ import LineReader from "line-reader"
 import { fileURLToPath } from "url"
 
 const pool = new pg.Pool({
-  connectionString: "postgres://postgres:password@localhost:5432/<yourDBnameHere>" })
+  connectionString: "postgres://postgres:password@localhost:5432/animal_shelter_development"
+})
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+const petTypesPath = path.join(__dirname, "../../petTypes.txt")
 
 class Seeder {
   static async seed() {
-    // your seeder code here
+    LineReader.eachLine(petTypesPath, async (line, last, done) => {
+      const [name, img_url, description] = line.split(";")
+      // build our SQL query string
```

Comment:
In general, we don't want to leave explanatory comments - well-written code is self-explanatory